doc-deploy-dev/action.yml

```diff
@@ -208,36 +189,33 @@ runs:
       with:
         level: "INFO"
         message: >
-          Generate an 'index.html' for redirection to the latest stable version
-          of the documentation. If no stable version has been released, a
-          redirection to the development documentation is generated.
+          Create an index.html page copied from index.html in the stable version.
+          Update all the local href and source links to point to the stable or dev version. 
 
-
-    - name: "Generate the redirection URL"
+    - name: "Use the latest 'index.html' in base index.html page"
       shell: bash
       run: |
-        if $(grep -q "(stable)" versions.json)
-        then
-          echo "Redirecting to the latest stable version..."
-          echo "REDIRECTION_URL=https://${{ inputs.cname }}/version/stable/" >> $GITHUB_ENV
+        if [[ -f 'version/stable/index.html' ]]; then
+          cp version/stable/index.html index.html
+        elif [[ -f 'version/dev/index.html' ]]; then
+          cp version/dev/index.html index.html
         else
-          echo "Redirecting to the latest development version..."
-          echo "REDIRECTION_URL=https://${{ inputs.cname }}/version/dev/" >> $GITHUB_ENV
+          echo "Error: The 'index.html' file does not exist." >&2
+          exit 1
         fi
 
-    - name: "Create the 'index.html' redirection file"
+    - name: "Update all local href and source links to version/stable"
       shell: bash
       run: |
-        echo "<!DOCTYPE html>" > index.html
-        echo "<meta charset=\"utf-8\">" >> index.html
-        if [ -z "${{ inputs.redirection-title }}" ]; then
-          echo "<title>Redirecting to ${{ env.REDIRECTION_URL }}</title>" >> index.html
+        # Determine the version directory based on the existence of stable index.html
+        if [[ -f 'version/stable/index.html' ]]; then
+          version_dir="version/stable"
         else
-          echo "<title>${{ inputs.redirection-title }}</title>" >> index.html
+          version_dir="version/dev"
         fi
-        echo "<meta name=\"description\" content=\"${{ inputs.redirection-description }}\">" >> index.html
-        echo "<meta http-equiv=\"refresh\" content=\"0; URL=${{ env.REDIRECTION_URL }}\">" >> index.html
-        echo "<link rel=\"canonical\" href=\"${{ env.REDIRECTION_URL }}\">" >> index.html
+        # Update URLs
+        sed -i "s#href=\"\([^h]\)#href=\"$version_dir/\1#g" index.html
+        sed -i "s#src=\"\([^h]\)#src=\"$version_dir/\1#g" index.html
```

Comment:
Please, use the same command in here as in the "stable" action.

Reply:
added.